code/WorkInProgress/rewardsLocker.dm

```diff
@@ -193,6 +193,51 @@
 			activator.set_clothing_icon_dirty()
 			M.wear_layer = MOB_BACK_LAYER_SATCHEL
 
+		else if (istype(activator.back, /obj/item/storage/backpack/studdedwhite) || activator.back.icon_state == "bp_studdedw")
```

Comment:
so cursed

Reply:
The entire satchel reward block of code is just... UTTERLY WRETCHED